src/app/cmdoptions.cpp

```diff
@@ -481,6 +484,10 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
         }
     }
 
+    if (QDir(QCoreApplication::applicationDirPath()).exists(DEFAULT_PORTABLE_MODE_PROFILE_DIR)) {
```

Comment:
Curly braces aren't needed in this if statement.
https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#e-single-statement-if-blocks